apps/server/src/modules/learnroom/controller/dto/index.ts

```diff
@@ -1,16 +1,17 @@
+export * from './course-export.body.params';
 export * from './course-import.body.params';
 export * from './course-metadata.response';
+export * from './course-room-element.url.params';
+export * from './course-room.url.params';
+export * from './course-sync.body.params';
 export * from './course.query.params';
 export * from './course.url.params';
+export { CreateCourseBodyParams } from './create-course-body.params';
```

Comment:
I'd say use the barrel export in order to have the whole file follow the same export convention

Reply:
Done ✅